plugins/woocommerce-admin/bin/modified-editsite-lock-unlock.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+var consentString = 'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.';
+
+if (window.wcSettings && window.wcSettings.wpVersion) {
```

Comment:
some whitespace issues in this file 😄

Reply:
Thank you for the catch 👍 

Updated in https://github.com/woocommerce/woocommerce/pull/40884/commits/2d3756a82ccffbde6a71644947124c86bc075a47